src/server/src/main/java/io/cassandrareaper/resources/ClusterResource.java

```diff
@@ -241,8 +241,8 @@ private Response addOrUpdateCluster(
     return Response.created(location).build();
   }
 
-  @Nullable // if cluster can't be found over jmx
-  private Cluster findClusterWithSeedHost(String seedHost, Optional<Integer> jmxPort) {
+  @Nullable // if cluster can't be found
+  public Cluster findClusterWithSeedHost(String seedHost, Optional<Integer> jmxPort) {
```

Comment:
trivial: ideally we should have changed the return type from `@Nullable` to `Optional<Cluster>` when making the method public.

Reply:
ok, done.
Will appear in my next force push.